.github/workflows/pull-request.yml

```diff
@@ -139,16 +139,3 @@ jobs:
           PLATFORM: ${{ matrix.platform }}
         run: ./scripts/docker-qemu.sh
           ../sof.git/scripts/qemu-check.sh ${PLATFORM}
-
-  yamllint-gh:
```

Comment:
@marc-hb im pretty sure it didnt actually run because it is looking for this job, we need update the protected branches matches

Reply:
I'm not sure why the summary did not update but for sure the check itself ran. Quoting https://github.com/thesofproject/sof/pull/4132/checks?check_run_id=2606149741:
```
Note: switching to 'refs/remotes/pull/4132/merge'.
  
  HEAD is now at ed6ce7a Merge b5b59e7ef0261560d51b1e164785dae62bcda41c into f510435a253373c806c49fbc32d876287bcbe801
```

b5b59e7ef0261 is the last commit you pushed.

Reply:
Yes that is the new yamllint job I added which is `codestyle / yamllint` which is also showing up below. It is stuck on `yamllint /.github/workflows/ ` which you can see is the name of the above deleted job. I think its stuck because the protected branch requires it.

Reply:
I keep looking but I don't see anything stuck sorry, I only see the status not updated in the summary.

( https://github.com/thesofproject/sof/runs/2607493012?check_suite_focus=true ran again fine and the status was not updated again)

Reply:
Sorry I get it now: missed the difference between the new yamllint check and the old yamllint check. Only the removed check is "stuck" which is maybe only because it's getting removed. I bet this will fix itself and just go away once this PR is merged.

Reply:
> Sorry I get it now: missed the difference between the new yamllint check and the old yamllint check. Only the removed check is "stuck" which is maybe only because it's getting removed and does not exist anymore. I bet this will fix itself and just go away once this PR is merged.

Looks like it "fixed itself", the old yamllint is now gone, see for instance #4211 and others. Let's assume github struggles a bit during the transition when renaming some Actions files.

Reply:
I bet it disappeared when i removed the rule to require it. Basically protected branch logic stopped looking for something that no longer exists